src/main/java/sirius/db/mixing/annotations/RemoveWhiteSpaces.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.db.mixing.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Marks a string property to remove any white-spaces.
+ * <p>
+ * All white-spaces of the property's value will be removed before it is written to the database.
+ */
+@Documented
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface RemoveWhiteSpaces {
```

Comment:
Whitespace is a single word: https://en.wikipedia.org/wiki/Whitespace_(programming_language)

Reply:
Renamed. I thought on the action, since we rename *all* of them.